coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -3939,10 +3939,18 @@ def dim(context, node):
 def min(context, node):
     inputs = _get_inputs(context, node)
 
-    if len(inputs) == 3:
+    # mimic functionality from https://pytorch.org/docs/stable/generated/torch.min.html
+    if len(inputs) == 1:
+        value = mb.reduce_min(x=inputs[0], axes=None, name=node.name)
+        context.add(value)
+    elif len(inputs) == 2 and inputs[0].shape == inputs[1].shape:
+        value = mb.minimum(x=inputs[0], y=inputs[1], name=node.name)
+        context.add(value)
+    else:
+        assert(len(inputs) <= 3)
         _input = inputs[0]
         dim = inputs[1].val
-        keepdim = inputs[2].val
+        keepdim = inputs[2].val if len(inputs) == 3 else False
```

Comment:
I don't understand the change on this line. I think if execution gets here len(inputs) == 3 will always be True.

Reply:
Similar to the explanation above, if we reach here we could have been given inputs of length 2 of the form (input: Tensor, dim: int) or (input:Tensor, dim:int, keepdim:bool).